examples/text-generation/run_generation.py

```diff
@@ -193,15 +193,23 @@ def main():
     torch.distributed.barrier()
 
     # Initialize the model
-    model = deepspeed.init_inference(
-        model,
-        mp_size=world_size,
-        dtype=torch.bfloat16,
-        injection_policy={BloomBlock: ("self_attention.dense", "mlp.dense_4h_to_h")},
-        checkpoint=checkpoints_json,
-        args=args,
-        enable_cuda_graph=args.use_hpu_graphs,
-    )
+    if deepspeed.version >= "0.7.7+":
```

Comment:
I don't think we need this actually. Optimum Habana is going to rely on Habana's DeepSpeed 1.9 after [this PR](https://github.com/huggingface/optimum-habana/pull/193) is merged, so let's just wait for it and then we can merge your PR without checking the version of DeepSpeed.